templates/setupHaxeProject.yaml

```diff
@@ -0,0 +1,21 @@
+name: Simple-TypeScript
+description: This setup is for a simple Haxe TypeScript project. It includes a src dir, Compile.hxml and a Main.hx script.
+steps:
+  - name: "Create folders"
+    run: mkdir src
+  - name: "Create .gitignore and its contents"
+    run: touch .gitignore ;
+            echo 'bin' >> .gitignore ;
+  - name: "Create Main.hx and its contents"
+    run: touch src/Main.hx ;
+          echo "class Main {" >> src/Main.hx ;
+          echo "  static functon main() {" >> src/Main.hx ;
+          echo "    trace(\"hello, world\");" >> src/Main.hx ;
+          echo "  }" >> src/Main.hx ;
+          echo "}" >> src/Main.hx
```

Comment:
Please create a file in `./templates/src/` called `hello-world.hx` and paste its content in there. Your step can be replaced with `cp $PROJECT_DIR/templates/src/hello-world.hx ./src/Main.hx`

Reply:
Fixed!